Templates/AbstractAjaxTemplate/AbstractAjaxTemplate.php

```diff
@@ -389,20 +389,20 @@ protected function splitRowsByMultivalueFields(array $rows, DataSheetInterface $
      *
      * @see \exface\Core\CommonLogic\AbstractTemplate::processRequest()
      */
-    public function processRequest($page_id = NULL, $widget_id = NULL, $action_alias = NULL, $disable_error_handling = false)
+    public function processRequest($page_alias = NULL, $widget_id = NULL, $action_alias = NULL, $disable_error_handling = false)
     {
         // Look for basic request parameters
-        $called_in_resource_id = $page_id ? $page_id : $this->getRequestPageId();
+        $called_in_resource_alias = $page_alias ? $page_alias : $this->getRequestPageAlias();
```

Comment:
Hier bin ich mir nicht sicher. Ist es wirklich notwendig, den Alias in den Requests zu schicken? Wäre es nicht einfacher es bei der ID zu belassen? Was waren deine Beweggründe?

Reply:
Die UID wird jetzt praktisch nur noch innerhalb der UiPage und im ModxCmsConnector verwendet. Überall sonst werden Aliase verwendet, weshalb sie jetzt auch in den Requests verwendet werden.